viewing_party/party.py

```diff
@@ -0,0 +1,170 @@
+# Wave 1
+
+def create_movie(title, genre, rating):
+    create_movie_dict = {
+        "title": title,
+        "genre": genre,
+        "rating": rating
+    }
+    if title and genre and rating:
+      return create_movie_dict 
+    else:
+      return None
+
+def add_to_watched(user_data, movie):
+    watched_list = user_data["watched"]
+    watched_list.append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    add_watchlist = user_data["watchlist"]
+    add_watchlist.append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    for movie in user_data['watchlist']:
+        if title == movie['title']:
+            user_data['watchlist'].remove(movie)
```

Comment:
even though remove works great here, be careful using it in a for loop. It can cause side effects. Here is a link that explains it more with examples https://thispointer.com/python-remove-elements-from-a-list-while-iterating/